src/sage/combinat/triangles_FHM.py

```diff
@@ -667,6 +667,25 @@ def m(self):
         polym = step.numerator()
         return M_triangle(polym, variables=(x, y))
 
+    def parabolic(self):
+        """
+        Return a parabolic version of the F-triangle.
+
+        This is obtained by replacing the variable `y` by `y-1`.
+
+        EXAMPLES::
+
+            sage: from sage.combinat.triangles_FHM import H_triangle
+            sage: x, y = polygens(ZZ,'x,y')
+            sage: H_triangle(1+x*y).f()
+            F: x + y + 1
+            sage: _.parabolic()
+            F: x + y
+        """
+        x, y = self._vars
+        polyf = self._poly(y=y - 1)
+        return F_triangle(polyf, variables=(x, y))
```

Comment:
```suggestion
        y = self._vars[-1]
        polyf = self._poly(y=y - 1)
        return F_triangle(polyf, variables=self._vars)
```
Can you also add a test when the variables are not `x,y`; say `a,b`? I am worried about the substitution line. It might need to be `self._poly(**{str(y): y})`.

Reply:
ah, right.. I am afraid that the whole file is not robust with respect to the use of other variable names. There are many substitutions where I use `x=..` and / or `y=...`.

Should I fix them all here ?

Reply:
Up to you. At least for this method we should do it here. Probably best to do it en masse here though...